Tasks/InstallAppleCertificateV2/preinstallcert.ts

```diff
@@ -75,7 +75,8 @@ async function run() {
         }
         tl.setTaskVariable('APPLE_CERTIFICATE_KEYCHAIN', keychainPath);
 
-        await sign.installCertInTemporaryKeychain(keychainPath, keychainPwd, certPath, certPwd, true);
+        const setUpPartitionIdACLForPrivateKey = tl.getBoolInput('setUpPartitionIdACLForPrivateKey', false);
+        await sign.installCertInTemporaryKeychain(keychainPath, keychainPwd, certPath, certPwd, true, setUpPartitionIdACLForPrivateKey);
```

Comment:
nit: Just to improve readability 😄 

```suggestion
        const useKeychainIfExists: boolean = true;
        await sign.installCertInTemporaryKeychain(keychainPath, keychainPwd, certPath, certPwd, useKeychainIfExists, setUpPartitionIdACLForPrivateKey);
```